db/database.rb

```diff
@@ -0,0 +1,16 @@
+require 'sequel'
+require 'pg'
+require 'singleton'
+
+class Database
```

Comment:
In the future we can refactor this class a bit and use a Rake task to automate some of its functionality.